data/campaigns/Under_the_Burning_Suns/units/quenoth/Mystic.cfg

```diff
@@ -28,17 +30,17 @@
         #textdomain wesnoth-utbs
         type=impact
         range=melee
-        damage=4
+        damage=3
         number=2
         range=melee
         icon=attacks/druidstaff.png
     [/attack]
     [attack]
         name=sand
-        description= _"sand"
+        description= _"sandstorm"
```

Comment:
I don't mind changing this to sandstorm for higher levels but it seems like it'd be neat to keep it as just plain sand for the lvl 1 unit.

Reply:
Maybe sandwind as moderated name? :P I honestly just wanted to be consistent and I liked the name way more.

Reply:
Nah then keep it.